src/test/java/com/itextpdf/rups/RupsConfigurationTest.java

```diff
@@ -128,7 +128,9 @@ public void setSystemLookAndFeel() {
         RupsConfiguration.INSTANCE.setLookAndFeel(laf);
         RupsConfiguration.INSTANCE.saveConfiguration();
         String lookAndFeel = RupsConfiguration.INSTANCE.getLookAndFeel();
-        Assertions.assertTrue(lookAndFeel.contains("javax.swing.plaf") || lookAndFeel.equals("com.sun.java.swing.plaf.windows.WindowsLookAndFeel"));
+        Assertions.assertTrue(lookAndFeel.contains("javax.swing.plaf")
+            || lookAndFeel.contains("java.swing.plaf")
```

Comment:
Hmmm, looking at it again, I guess we don't need the WinBlows-specific check as it'll pass my broader check just for java.swing.plaf anyway